File: src/python/test/TestLatin.py
import unittest
from Utils import Utils 
from Refi import Refi


class TestLatin(unittest.TestCase):

    def setUp(self):
        self.refi = Refi()
        self.utils = Utils()


    def test_(self):

        expression = self.refi.toRegEx("latin")
        self.utils.assertRegexIs("\\p{IsLatin}", expression)
        self.utils.assertRegexMatchesText(expression, "A")
        self.utils.assertRegexDoesntMatchText(expression, "\\u0393")

    def runTest(self):
        pass  

if __name__ == '__main__':
    unittest.main()    
    

File: src/test/java/koorkah/TestLatin.java
package koorkah;

import org.junit.jupiter.api.Test;

class TestLatin extends BaseT
{

    @Test
    void test()
    {
        String expression = koorkah.toRegEx("latin");
        assertRegexIs("\\p{IsLatin}", expression);
        assertRegexMatchesText(expression, "A");
        assertRegexDoesntMatchText(expression, "\\u0393");
    }

}
